packages/definitions/src/integration.ts

```diff
@@ -176,6 +176,25 @@ export const integrationDefs = {
     iconUrl: "https://cdn.jsdelivr.net/gh/homarr-labs/dashboard-icons@master/svg/ntfy.svg",
     category: ["notifications"],
   },
+  // This integration only returns mock data, it is used during development (but can also be used in production by directly going to the create page)
```

Comment:
1. How does it make sense to use this in production?
2. Why allow users to directly create it and bypass the check?

Reply:
1. For example if you need to reproduce an issue somebody is having (visually) of a widget where you don't have the service that is needed for the integration, so it's more thought for us to debug stuff. And it also allows users to still view how a widget looks if they really care.
2. As I said, I just want to hide it to prevent confusion